plot_app/configured_plots.py

```diff
@@ -470,46 +470,30 @@ def generate_plots(ulog, px4_ulog, db_data, vehicle_data, link_to_3d_page,
     plot_flight_modes_background(data_plot, flight_mode_changes, vtol_states)
     if data_plot.finalize() is not None: plots.append(data_plot)
 
-
-    # actuator outputs 0: Main
-    data_plot = DataPlot(data, plot_config, 'actuator_outputs',
-                         y_start=0, title='Actuator Outputs (Main)', plot_height='small',
-                         changed_params=changed_params, x_range=x_range)
-    num_actuator_outputs = 16
-    if data_plot.dataset:
-        max_outputs = np.amax(data_plot.dataset.data['noutputs'])
-        if max_outputs < num_actuator_outputs: num_actuator_outputs = max_outputs
-    data_plot.add_graph(['output['+str(i)+']' for i in range(num_actuator_outputs)],
-                        [colors8[i % 8] for i in range(num_actuator_outputs)],
-                        ['Output '+str(i) for i in range(num_actuator_outputs)], mark_nan=True)
-    plot_flight_modes_background(data_plot, flight_mode_changes, vtol_states)
-
-    if data_plot.finalize() is not None: plots.append(data_plot)
-
-    # actuator outputs 1: AUX
-    data_plot = DataPlot(data, plot_config, 'actuator_outputs',
-                         y_start=0, title='Actuator Outputs (AUX)', plot_height='small',
-                         changed_params=changed_params, topic_instance=1,
-                         x_range=x_range)
-    num_actuator_outputs = 16
-    # only plot if at least one of the outputs is not constant
-    all_constant = True
-    if data_plot.dataset:
-        max_outputs = np.amax(data_plot.dataset.data['noutputs'])
-        if max_outputs < num_actuator_outputs: num_actuator_outputs = max_outputs
+    actuator_output_plots = [(0, "Actuator Outputs (Main)"), (1, "Actuator Outputs (AUX)"), (2, "Actuator Outputs (EXTRA)")]
+    for topic_instance, plot_name in actuator_output_plots:
+
+        data_plot = DataPlot(data, plot_config, 'actuator_outputs',
+                            y_start=0, title=plot_name, plot_height='small',
+                            changed_params=changed_params, topic_instance=topic_instance, x_range=x_range)
+        num_actuator_outputs = 16
+        # only plot if at least one of the outputs is not constant
+        all_constant = True
+        if data_plot.dataset:
+            max_outputs = np.amax(data_plot.dataset.data['noutputs'])
+            if max_outputs < num_actuator_outputs: num_actuator_outputs = max_outputs
 
         for i in range(num_actuator_outputs):
```

Comment:
Indentation is off here
```suggestion
            for i in range(num_actuator_outputs):
```

Reply:
Solved.